tracer.go

```diff
@@ -116,7 +114,7 @@ type Tracer interface {
 
 // StartSpanOptions allows Tracer.StartSpan() callers and implementors a
 // mechanism to override the start timestamp, specify Span References, and make
-// Tags available at Span start time.
+// a single Tag or multiple Tags available at Span start time.
```

Comment:
Staging these updates here:
https://github.com/opentracing/opentracing-go/pull/94
I went with "specify one or more tags" to keep it simple, and I think that's clear enough.